broker/config/settings.yml

```diff
@@ -782,7 +782,7 @@ defaults: &defaults
           update_predecessors:
           - 'bc158c9a-7934-401e-94ab-057082a5073f'
           template: <%= base64_template('templates/blueprint-manifest.yml.ejs') %>
-          dashboard_url_template: <%= new Buffer('http://service-fabrik-broker.bosh-lite.com/manage/dashboards/director/instances/${instance_id}?planName=${plan.name}&serviceId=${plan.service.id}').toString('base64') %>
+          dashboard_url_template: <%= new Buffer('${spec.clusterId == 1 ? \'http://service-fabrik-broker.bosh-lite.com/manage/dashboards/director/instances/\'+metadata.name+\'?planId=\'+spec.planId+\'&serviceId=\'+spec.serviceId : \'\'}').toString('base64') %>
```

Comment:
I am little hesitant about two things here.

1. We are not passing context anymore. Are we missing out any attribute ? I guess no.
2. ABAP colleagues use this feature. They have to adapt when changing to this version as the keys are changed. 

Should we keep context and add `sfserviceinstance`'s `clusterId` as it's sub attribute ?

Reply:
Changed it such that status quo is maintained. Now you can access `sfserviceinstance` values inside template by using `instance` objejct
Example: `instance.spec.clusterId`